sormas-ui/src/main/java/de/symeda/sormas/ui/samples/PathogenTestController.java

```diff
@@ -276,16 +276,18 @@ private void handleAssociatedContact(
 		final boolean equalDisease = dto.getTestedDisease() == contact.getDisease();
 
 		Runnable callback = () -> {
+			if (!UserProvider.getCurrent().hasUserRight(UserRight.CONTACT_EDIT)) {
```

Comment:
The problem is that the pop-up will ask for creating a case out of the contact, which requires `CASE_CREATE` and `CONTACT_EDIT` rights. The clinician doesn't have these rights.
Probably we could show a different popup which just informs the user that this action would normally result in a new case, but for this he should contact someone who has the rights to do that and if he still whishes to continue adding the sample without creating a new case.

Reply:
@leventegal-she @vidi42 I agree that the dialog to change the sample result should definitely be shown, and that this needs to be done in this PR. And I also agree that we still need to handle the case creation stuff in some way. My suggestion would be the following:

If a user does not have the `CASE_CREATE` and/or `CONTACT_EDIT` user right, instead of showing the popup to create a case for the contact person, ask them whether they want to create a task that is automatically assigned to either the user assigned as the responsible user to the contact, or an eligible user in the contact's jurisdiction (i.e. probably any user that has both the `CASE_CREATE` and `CASE_RESPONSIBLE` user right, but we'd need to refine this). This would be a new task type, probably called something like `CREATE_CASE_FROM_SAMPLE`, that would be assigned to the contact and would probably need a link to the sample in question in the `creatorComment` field.

However, this can definitely be moved to a separate change request.

@MartinWahnschaffe Can you check whether this makes sense to you?

Reply:
Yes, a follow-up issue to create a task as a fallback makes sense. Until then it's fine to not show the popup to create a case.

Reply:
Follow-up ticket https://github.com/hzi-braunschweig/SORMAS-Project/issues/8897. I will merge the PR for now